tox.ini

```diff
@@ -29,6 +29,18 @@ extras =
 commands =
     pytest {posargs}
 
+[testenv:coverage]
+description = Invoke nile-coverage to generate coverage report
+skip_install = True
+setenv =
+    {[testenv]setenv}
+    CAIRO_PATH = src
```

Comment:
why is this required for coverage?

Reply:
This is related to [this](https://github.com/OpenZeppelin/cairo-contracts/pull/494/files#r1005024236)

We need to specify the cairo path for the compiler to resolve imports, so the VM can catch the correct files when generating the coverage report. This wasn't necessary when just testing because the imports were resolved then from the installed library, but this won't work with coverage because we need to cover local files, not installed files under python/lib.